projects/aca-shared/src/lib/components/document-base-page/document-base-page.spec.ts

```diff
@@ -92,8 +92,8 @@ class DocumentBasePageServiceMock extends DocumentBasePageService {
 class TestComponent extends PageComponent {
   node: any;
 
-  constructor(store: Store<AppStore>, extensions: AppExtensionService, content: DocumentBasePageService) {
-    super(store, extensions, content);
+  constructor() {
```

Comment:
That constructor seems redundant?

Reply:
Angular was still requiring me to call the super even if it is empty

Reply:
ok then fine. I thought it is redundant, because I checked out to your branch @DenysVuika and tried removing whole constructor at all (as it does nothing) and there was no errors in intellij at least: 
<img width="1662" alt="image" src="https://user-images.githubusercontent.com/115619721/236810168-1e08f803-da3e-4ad9-84ba-ea10c269c424.png">

But if angular itself complains about that then fine